### PR TITLE
coordinate parsing fixes

### LINKED ISSUE #667: GTP Cmd Error : Uppercase "PASS" causes "illegal move"
**Circumstance**
Linux version 4.9.0-8-amd64 (debian-kernel@lists.debian.org) (gcc version 6.3.0 20170516 (Debian 6.3.0-
18+deb9u1) ) #1 SMP Debian 4.9.130-2 (2018-10-27)
Python 3.5.3
------------------

I tested Minigo v3 (9 * 9) gtp on the CGOS server.
If "PASS" is displayed on the other side(Gnugo-3.7.10.-a1(1800), "illegal move" occurs and the cgosgtp-linux client is forcibly terminated. Though I try again, the same error message displayed and couldnt end the game. (Eventually lost the game by timeout)

http://www.yss-aya.com/cgos/9x9/SGF/2019/01/28/711010.sgf
http://www.yss-aya.com/cgos/9x9/SGF/2019/01/28/710995.sgf

19:03:12    C->E play W J7
19:03:12    E->C = true
19:03:12    C->E play B H5
19:03:12    E->C = true
19:03:12    C->E play W D1
19:03:12    E->C = true
19:03:12    C->E play B E1
19:03:12    E->C = true
19:03:12    C->E play W C3
19:03:12    E->C = true
19:03:12    C->E play B F5
19:03:12    E->C = true
19:03:12    C->E play W C2
19:03:12    E->C = true
19:03:12    C->E play B C8
19:03:12    E->C = true
19:03:12    C->E play W H9
19:03:12    E->C = true
19:03:12    C->E play B H1
19:03:12    E->C = true
19:03:12    C->E play W A3
19:03:12    E->C = true
19:03:12    C->E play B H3
19:03:12    E->C = true
19:03:12    C->E play W PASS
**19:03:12    E->C ? illegal move**
19:03:12    Error. Engine returns error error response.  Aborting
creeras@instance-1:~/minigo/app$ cd ..

It seems that there is an **error when the first character is capitalized**. How can I solve it?
(At Gtp sole mode)
----------------------------------

play w pass
__GTP_CMD_DONE__
= true

play b pass
__GTP_CMD_DONE__
= true

play b PASS
Traceback (most recent call last):
  File "/home/creeras/minigo/gtp_engine.py", line 171, in handle_msg
    _print_success(msg_id, handler(*args))
  File "/home/creeras/minigo/gtp_cmd_handlers.py", line 71, in cmd_play
    return self._player.play_move(coords.from_kgs(move))
  File "/home/creeras/minigo/coords.py", line 84, in from_kgs
    row_from_bottom = int(kgsc[1:])
ValueError: invalid literal for int() with base 10: 'ASS'
__GTP_CMD_DONE__
? illegal move

play w Pass
Traceback (most recent call last):
  File "/home/creeras/minigo/gtp_engine.py", line 171, in handle_msg
    _print_success(msg_id, handler(*args))
  File "/home/creeras/minigo/gtp_cmd_handlers.py", line 71, in cmd_play
    return self._player.play_move(coords.from_kgs(move))
  File "/home/creeras/minigo/coords.py", line 84, in from_kgs
    row_from_bottom = int(kgsc[1:])
ValueError: invalid literal for int() with base 10: 'ASS'
__GTP_CMD_DONE__
? illegal move

### PR DESCRIPTION
**Pull request by tommadams**
Fixes #667 and #640 

- Rename KGS -> GTP in Python code.
- SGF: parse `tt` as pass for board sizes <= 19 for both Python and C++ (I misread the spec and C++ previously only did this for board size == 19).
- GTP: parse uppercase `PASS` as pass in Python code (C++ already does this correctly).

Note that `oneoffs/prepare_bigquery.py` and `oneoffs/bigquery_moves_schema.json` still have references to `move_kgs` but I don't know if those are safe to change.

### CODE REVIEW

File: coords.py [hunk 1]
```diff
@@ -75,19 +75,19 @@ def to_sgf(coord):
...
+    row_from_bottom = int(gtpc[1:])
     return go.N - row_from_bottom, col
 
 
-def to_kgs(coord):
-    """Converts from a MiniGo coordinate to a KGS coordinate."""
+def to_gtp(coord):
+    """Converts from a MiniGo coordinate to a GTP coordinate."""
     if coord is None:
         return 'pass'
```
Comment (sethtroisi) [inline coords.py]: should this be PASS?

File: coords.py [hunk 2]
```diff
@@ -75,19 +75,19 @@ def to_sgf(coord):
     return _SGF_COLUMNS[coord[1]] + _SGF_COLUMNS[coord[0]]
 
 
-def from_kgs(kgsc):
-    """Converts from a KGS coordinate to a MiniGo coordinate."""
-    if kgsc == 'pass':
+def from_gtp(gtpc):
+    """Converts from a GTP coordinate to a MiniGo coordinate."""
+    gtpc = gtpc.upper()
+    if gtpc == 'PASS':
```
Comment (sethtroisi) [inline coords.py]: I appreciate that you added an extra test for both cases :)

**Comment by amj** [comment]
@brilee re: the bigquery script changes.

File: coords.py (hunk 1 above)
Comment (tommadams) [reply coords.py]: The GTP spec uses lowercase for pass in its examples, so lets follow that convention.

**Pull request merged**